salt/modules/ipset.py

```diff
@@ -307,7 +319,7 @@ def check_set(set=None, family='ipv4'):
     return True
 
 
-def add(set=None, entry=None, family='ipv4', **kwargs):
+def add(setname=None, entry=None, family='ipv4', **kwargs):
```

Comment:
Could you clarify the reasoning for changing this, please?

Reply:
In code there is redefinition of class set, I want to use set collection in code, but there is another option if this is problem.
```if len(set(['skbmark', 'skbprio', 'skbqueue']) & set(kwargs.keys())) > 0:```
This is simply if any of add options is in kwargs then check if it is defined skbinfo as create option.

Reply:
Ahhh, yes. Great point! Makes perfect sense. Thanks.

Reply:
@cachedout Hi, I have made mistake in this pull request with state absent. I have fix it in #44660. Please merge.
But I found another problem with ipset module when there is create option "counters" active. It does not delete entries if it is created with options. This is because "packets" and "bytes" shows in ipset output. And same is with adding entries it adds it multiple times when this option is present.